task3/frontpage.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="frontpage.css"> 
+<head>
+   <!-- <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
+    <title>NTW</title>
+</head>
+<body  >
+    <header class="blok "  >
+        
+        <div class="d1 box0"></div>
+        
+        <div class="d2 box1 ">
+        
+            <p class="avtor">NTW Сальдюн Андрей 2 курс 7 группа  asaldyun@gamil.com  v.0.0</p>
+            
+            <form action="sing in"> 
+                <button class="buttonsin" type="Sing in">Sing in</button>
+            </form>  
+        
+            <img class="prphoto" src="image/person-icon.png"
+            alt="profil photo">
+        
+            <form action="sub"> 
+                <button class="buttonsub" type="submit">Subscribe</button>
+            </form> 
+
+            <p class="prfinfo" > profill info</p>
+
+            <form >
```

Comment:
All right filter...
that's right that it is form
but two details regarding inputs here
1) p means paragraph (of text), input field is not the text...
2) do not use inline styles or legacy ability to define some properties trough attributes (like width) use external CSS.
3) Inputs that holds information are usually called with the name attribute,so form will now which key  to use to define user input.
4) For wrapping inputs use semantic less div or label in case you also want provide the label for the field.